components/MenuEditorials/MenuItem.js

```diff
@@ -46,17 +53,17 @@ const MenuItem = ({color, content, groupTitleProps, removeBorders}) => {
 
   const Group = () => (
     <div>
-      <GroupTitleContainer id={contentId} onClick={handleItemClick} removeBorders={removeBorders} $color={itemColor} >
+      <GroupTitleContainer id={contentId} onClick={handleItemClick} pl={pl} removeBorders={removeBorders} $color={itemColor}>
         <GroupTitle/>
-        {open ? <ExpandLessIcon/> : <ExpandMoreIcon/>}
+        {open ? <ExpandLessIcon color={itemColor}/> : <ExpandMoreIcon color={itemColor}/>}
       </GroupTitleContainer>
       <GroupContent>
         {open && 
-          map(subitems, ({path, name}, key) => (
+          map(subitems, ({name}, key) => (
```

Comment:
Seguindo a sugestão de cima

```suggestion
          map(subitems, ({name, path}, key) => (
```